pkg/kernel/patches-5.10.x/0014-net-fec-Silence-M5272-build-warnings.patch

```diff
@@ -0,0 +1,72 @@
+From 0eb93b443100b98e7e2fe0e1c8c32673fdb6e952 Mon Sep 17 00:00:00 2001
+From: Geert Uytterhoeven <geert@linux-m68k.org>
+Date: Tue, 2 Feb 2021 14:06:50 +0100
+Subject: [PATCH 14/21] net: fec: Silence M5272 build warnings
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+If CONFIG_M5272=y:
+
+    drivers/net/ethernet/freescale/fec_main.c: In function ‘fec_restart’:
```

Comment:
In the PR's description, it's explained the options of get drivers from Linux 6.x or from vendor's BSP. It's not clear from where these drivers are coming from. In the commit message, it's also mentioned that ethernet driver fixes are introduced. It would be great if you pointed out in the commit message from which repo drivers were cherry-picked (when it's the case), and for those that fixes (changes) were introduced, please, give more details so we can keep track of them.

Reply:
Make sence

Reply:
I add all links to commit with patches